### PR TITLE
v1.2.0 proposal

### PR DESCRIPTION
**Pull request by jjelosua**
Hi all,
* @coreyhaines: Taking as a base your PR with customEvents I have moved the pym-initialized event to `pym` and also added a pym-loaded event on the `pym-loader`.
* @quickhand: #146 to make  safe universal rendering is on this PR too.
* @AlainVagner: #147 to override parentUrl is on this PR too.
* @benlk: #120 to make the parentUrl configurable is on this PR too.

I have included more tests to add coverage for the new functionality and examples to run locally.

@coreyhaines, My idea is to write some documentation and publish next week, let me know if you see any issues.

### CODE REVIEW

File: dist/pym.v1.js [hunk 1]
```diff
@@ -128,7 +142,7 @@
      * @instance
      * @method autoInit
      */
-    lib.autoInit = function() {
+    lib.autoInit = function(ignoreEvent) {
```
Comment (coreyhaines) [inline dist/pym.v1.js]: Name would be more clear if something like `doNotRaiseEvents`
`ignoreEvents` implies that it is ignoring events from another place

File: dist/pym-loader.v1.js [hunk 2]
```diff
@@ -7,15 +7,32 @@
...
+    /**
     * Initialize pym instances if Pym.js itself is available
     *
     * @method initializePym
     * @instance
     *
     * @param {String} pym Pym.js loaded library.
     */
-    var initializePym = function(pym) {
+    var initializePym = function(pym, ignoreEvent) {
```
Comment (coreyhaines) [inline dist/pym-loader.v1.js]: Name would be more clear if something like doNotRaiseEvents
ignoreEvents implies that it is ignoring events from another place

**Comment by coreyhaines** [comment]
So, we're leaving the loading events on the pym-loader, it appears? Just adding a pym-initialized to pym?

File: dist/pym.v1.js (hunk 1 above)
Comment (jjelosua) [inline dist/pym.v1.js]: Sounds good

**Comment by jjelosua** [comment]
@coreyhaines Yeah, I thought that leaving the pym-loaded on the loader made sense....any issue with that?

**Comment by coreyhaines** [comment]
Other than the comments about the name, this looks 👍

**Pull request merged**